test/Bin/GameTest/GameTestMain.cpp

```diff
@@ -29,13 +31,21 @@ class GameThread {
 
         initDataPath(options.dataPath);
 
-        _config = std::make_shared<GameConfig>("openenroth.ini");
+        // stop tests overwriting gameplay config
+        if (std::filesystem::exists(MakeDataPath("openenroth.ini"))) {
+            std::filesystem::copy(MakeDataPath("openenroth.ini"), MakeDataPath("openenroth_test.ini"));
+        }
```

Comment:
I don't think we need to copy `openenroth.ini`, we're overwriting all the config values anyway.